### DISCUSSION TITLE
Cannot create new gene lists (i.e. seems like only a limited number of gene lists can be created/added)

### DISCUSSION THREAD

**Discussion by Noma-M** [broadinstitute/seqr#2708] [Q&A]
**Describe the bug**
Trying to create more than one gene list with each list representing a targeted gene panel. Thirteen different gene lists were successfully added. As more gene lists are being added nothing pops up under the "my gene lists" page. 
It also seems like each user belonging to main project can only add a limited number of gene lists. For example, of the thirteen gene lists already created one user created 8 lists and could not create more. The 5 remaining gene lists were added by other users.

**Link to page(s) where bug is occurring**
Add Gene Lists on seqr

**Scope of the bug**
Gene lists on the main workspace

**Screenshots**
![image](https://user-images.githubusercontent.com/77622537/167416111-d853d8b3-6ceb-4390-88f9-8231314525d4.png)
may access this page and select for "Create new gene list"

![image](https://user-images.githubusercontent.com/77622537/167416408-092b5f51-a347-426c-a9a7-799dde70aa12.png)
may successfully add details for the gene list and select submit

However, after submitting nothing pops up on this page with the newly added gene list:
![image](https://user-images.githubusercontent.com/77622537/167416111-d853d8b3-6ceb-4390-88f9-8231314525d4.png)

**Comment by hanars** [comment]
Hi Nora,

There is no limit to the number of gene lists that can be added in seqr, or to the number of lists a single user can add. This sounds like a bug in your installation. At this point in time, can other users still add gene lists and there is one user who can not? Or can no user add gene lists anymore? When did this issue start occurring for you?

It would also be helpful if you could 
1) Attach the logs for the seqr container (`docker-compose logs seqr`)
1) Attach a copy of the docker-compose file you are currently running
1) Confirm when was the last time you updated your seqr image

**Comment by Noma-M** [comment]
Good day,

thank you for your response.

No user in our group can add gene lists.
We are using seqr on AnVIL/Terra, i.e. not a local installation so there
will be no container logs.

Kind regards,
Noma

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by hanars** [comment]
There appears to be a bug in seqr where if you provide invalid gene names it exits without creating the list instead of showing you the error. I have created https://github.com/broadinstitute/seqr/issues/2711 to track the work for fixing this issue.

**Comment by hanars** [comment]
Hi Nora, this bug is now fixed in seqr. Please give it a try and let me know if your users are still experiencing the issue

**Comment by Noma-M** [comment]
Good day, thank you. It has been fixed. Users are able to add gene lists
with an error notification that specifies the invalid gene name(s).

Many thanks!
